### ISSUE TITLE
realsr run error vkQueueSubmit failed -4

### ISSUE THREAD

**Issue by powerful001** [nihui/realsr-ncnn-vulkan#20]
error : vkQueueSubmit failed -4

![屏幕截图 2021-02-25 225116](https://user-images.githubusercontent.com/66508520/109172014-4845f280-77bd-11eb-96b6-02363ee014be.png)
Windows 10
gpu: 860m 
cpu: i7-4710
memory 16GB

**Comment by virdiignis** [comment]
Hi, did you find a solution?

**Comment by EdePC** [comment]
Quite a lot of VRAM/RAM required, use -m to decrease the amount of VRAM required, for example:

`realsr-ncnn-vulkan.exe -i image.jpg -o imagex4.jpg -t 100`

Use 100, 150 or more depending on the amount of VRAM to use. My GTX 960 not working with 200, but yes with 150, 100 or less

**Comment by karubn** [comment]
I am getting the same error, what is the solution?

**Comment by jbrown1597** [comment]
same

**Comment by karubn** [comment]
> Oldukça fazla VRAM/RAM gerekiyor; gerekli VRAM miktarını azaltmak için -m kullanın, örneğin:
> 
> `realsr-ncnn-vulkan.exe -i image.jpg -o imagex4.jpg -t 100`
> 
> Kullanılacak VRAM miktarına bağlı olarak 100, 150 veya daha fazlasını kullanın. GTX 960'ım 200 ile çalışmıyor ancak 150, 100 veya daha azıyla evet

Yes friends, the solution is related to the amount of RAM to be used, as our edcpc friend said, you need to set -t 100. For example, for Geforce GT 750m, this value works for me -t 55.
realsr-ncnn-vulkan.exe -i "%~dpnx1" -o "%~dpn1_Upscaled.png" -t 55 -g 0 -m models-DF2K_JPEG

**Comment by powerful001** [comment]
realsr-ncnn-vulkan.exe -i image.jpg -o imagex4.jpg -t num
Adding the num to the number before the error is the optimal solution
